t/lib/server.lua

```diff
@@ -482,6 +482,11 @@ function _M.google_logging_token()
     }))
 end
 
+function _M.casdoor_fake_access_token_api()
```

Comment:
We can put fake api in the test directly, like:
https://github.com/apache/apisix/blob/54ffdeed114bdb4f63ff72c8a867377d122a6b84/t/plugin/clickhouse-logger.t#L36